### PR TITLE
Added private blog and htaccess

### PR DESCRIPTION
**Pull request by m3rcuriel**

### CODE REVIEW

File: app/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{% for post in site.posts limit: 5 %}
+	<div class="post-preview">
+		<h2><a class="post-title" href="{{ post.url }}">{{ post.title }}</a></h2>
+		<span class="post-date">{{ post.date | date: "%B %d, %Y" }}</span>
+		<div class="post-content">
+			{{ post.content | split:'<!--break-->' | first }}
+		</div>
+ 
+		{% if post.content contains '<!--break-->' %}
+			<a class="read-more" href="{{ post.url }}">read more</a>
```
Comment (trotod) [inline app/blog/index.html]: Jekyll [provides a `post.excerpt` variable][0] that splits the post based on an `excerpt_separator` site variable. Or an `excerpt` can be set in the YAML front matter. So no need to split by `<!--break-->`, I guess.

[0]: http://jekyllrb.com/docs/posts/#post-excerpts

**Comment by trotod** [comment]
Github does not support .htaccess files (I think), so where are we hosting this?

Also, PR #7 has some extra things we could add.

File: app/blog/index.html (hunk 1 above)
Comment (m3rcuriel) [inline app/blog/index.html]: Can you get on gmail <3

**Comment by m3rcuriel** [comment]
We've never been hosting this on github. Get on gmail for video chat + shovik, akhil

**Pull request merged**